src/utils/deprecations.ts

```diff
@@ -8,3 +8,4 @@ export const noStringOperators = deprecate(noop, 'String based operators are dep
 export const noBoolOperatorAliases = deprecate(noop, 'A boolean value was passed to options.operatorsAliases. This is a no-op with v5 and should be removed.', 'SEQUELIZE0004');
 export const noDoubleNestedGroup = deprecate(noop, 'Passing a double nested nested array to `group` is unsupported and will be removed in v6.', 'SEQUELIZE0005');
 export const unsupportedEngine = deprecate(noop, 'This database engine version is not supported, please update your database server. More information https://github.com/sequelize/sequelize/blob/main/ENGINE.md', 'SEQUELIZE0006');
+export const useErrorCause = deprecate(noop, 'The "parent" and "original" properties in Sequelize errors have been replaced with the native "cause" property. Use that one instead.', 'SEQUELIZE0007');
```

Comment:
I see that 'parent' still works, but 'original' is removed. Is that on purpose?

Reply:
I'm not quite sure what the state of `original` is.
Prior to this PR, it was already not possible to pass an option called `original`, its value was always equal to `parent`

The deprecation warning mentions it because accessing the `original` property will log the warning too: https://github.com/sequelize/sequelize/pull/14318/files#diff-e86925093ea986c091226dc2f7e5af9b4c604ee4fa5c65f0795b7a4876eb16fdR44